documentation/anonymization.markdown

```diff
@@ -1,79 +1,75 @@
 
 ### Profile: full
 
-The `full` profile will anonymize everything, while keeping the shape of data as previously mentioned. This includes: node names, resource types, resource titles, parameter names, values, any log messages, file names and file lines. The result should be a completely anonymized data set. Report metrics under the `resources` and `events` categories are left intact, as these can be inferred from the rest of the data, but names of metrics under the `time` category are anonymized as resource types.
+The `full` profile will anonymize all data (including node names, resource types, resource titles, parameter names, values, any log messages, file names, and file lines) while retaining the data set's shape. The result is a completely anonymized data set. Report metrics under the `resources` and `events` categories are left intact, as these can be inferred from the rest of the data, but names of metrics under the `time` category are anonymized as resource types.
 
 This is useful if you are really concerned about limiting the data you expose, but provides the least utility for the consumer depending on the activity they are trying to test.
 
 ### Profile: moderate
 
-The `moderate` profile attempts to be a bit smarter about what it anonymizes and is the recommended one for most cases. It operates different depending on the data type:
+The `moderate` profile attempts to be a bit smarter about what it anonymizes and is **the recommended profile for most cases**. It sorts and anonymizes data by data type:
 
-* node name: is anonymized by default always
-* resource type name: the core types that are built-in to Puppet are not anonymized, including some common types from the modules: stdlib, postgresql, rabbitmq, puppetdb, apache, mrep, f5, apt, registry, concat and mysql. Any Puppet Enterprise core types names are also preserved. The goal here is to anonymize any custom or unknown resource type names as they may contain confidential information.
-* resource titles: all titles are anonymized expect for those belonging to Filebucket, Package, Service and Stage.
-* parameter names: are never anonymized
-* parameter values: everything is anonymized except for the values for `provider`, `ensure`, `noop`, `loglevel`, `audit` and `schedule`.
-* report log messages: are always anonymized
-* file names: are always anonymized
-* file numbers: are left as they are
-* log messages: are always anonymized
-* metrics: metric names in the `time` category are anonymized as resource
-  types.
+* Node name: is anonymized by default.
+* Resource type name: the core types that are built into Puppet are not anonymized, including some common types from the modules: `stdlib`, `postgresql`, `rabbitmq`, `puppetdb`, `apache`, `mrep`, `f5`, `apt`, `registry`, `concat`, and `mysql`. Any Puppet Enterprise core types names are also preserved. The goal here is to anonymize any custom or unknown resource type name, as these may contain confidential information.
```

Comment:
Any Puppet Enterprise core types names are also preserved.

"types" feels like it should be type, but I'm not sure if that's a PL technical term.

The goal here is to anonymize any custom or unknown resource type name, as these may contain confidential information.

"names" has been changed to "name". That feels odd to me, but maybe it was intentional?